freeipa/src/main/java/com/sequenceiq/freeipa/flow/freeipa/user/handler/SetPasswordHandler.java

```diff
@@ -0,0 +1,42 @@
+package com.sequenceiq.freeipa.flow.freeipa.user.handler;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.freeipa.client.FreeIpaClient;
+import com.sequenceiq.freeipa.flow.freeipa.user.event.SetPasswordRequest;
+import com.sequenceiq.freeipa.flow.freeipa.user.event.SetPasswordResult;
+import com.sequenceiq.freeipa.service.FreeIpaClientFactory;
+
+import reactor.bus.Event;
+
+@Component
+public class SetPasswordHandler implements FreeIpaClientEventHandler<SetPasswordRequest> {
+
```

Comment:
This class should implement the `com.sequenceiq.flow.reactor.api.handler.EventHandler`

Reply:
done